dev/com.ibm.ws.jdbc_fat_v43/test-applications/d43driver/src/org/test/d43/jdbc/D43Handler.java

```diff
@@ -63,6 +63,17 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
                 return this;
         }
 
+        //overriding default impls for the below methods to show they are being properly delegated to the driver
```

Comment:
Why did we override the default implementation with something that is less functional and less correct?  If we only need to show that our wrapper is being used, we could do that by having the test application first close the Statement and then attempting the operation, expecting the exception message to start with the typical DSRA####

Reply:
We can update the test to do that, although I wasn't particularly concerned with how "correct" this driver is.  It will make the test cases slightly more convoluted, but I guess that is ok.

Reply:
Given that other changes are being made anyway, I think we should fix this.  We should try to avoid getting in the way of already-correct implementation that the driver or JDBC spec default impl provides so that test applications can use all of the JDBC methods on the driver as naturally as possible.